src/context.c

```diff
@@ -563,6 +563,12 @@ const RaucContext *r_context(void)
 	return context;
 }
 
+void r_context_install_info_free(RContextInstallationInfo *info)
+{
+	/* contains only reference to existing bundle instance */
+	info->mounted_bundle = NULL;
+}
```

Comment:
Don't we need to free the struct itself here?

Reply:
Oh, yes. seems I missed that.

Reply:
OK, i'll add that as well.